# HG changeset patch
# User Rik <rik@octave.org>
# Date 1473722308 25200
#      Mon Sep 12 16:18:28 2016 -0700
# Node ID bb10d836751b64a4471febd9f21ef508fbe3da19
# Parent  76f2b0436423db7130074aaebb64bfd1052a6d70
NEWS: Review and update for 4.2 release.

* NEWS: Review and update for 4.2 release.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -1,25 +1,25 @@
 Summary of important user-visible changes for version 4.2:
 ---------------------------------------------------------
 
  ** The parser has been extended to accept, but ignore, underscore
     characters in numbers.  This facilitates writing more legible code
     by using '_' as a thousands separator or to group nibbles into bytes
     in hex constants.
 
-    Examples: 1_000_000 == 1e6 or 0xDE_AD_BE_EF
+    Examples: 1_000_000 == 1e6  or  0xDE_AD_BE_EF
 
  ** The parser has been extended to understand binary numbers which
     begin with the prefix '0b' or '0B'.  The value returned is Octave's
     default numeric class of double, not at unsigned integer class.
     Therefore numbers greater than flintmax, i.e., 2^53, will lose some
     precision.
 
-    Examples: 0b101 == 5 or 0B1100_0001 == 0xC1
+    Examples: 0b101 == 5  or  0B1100_0001 == 0xC1
 
  ** The default set of colors used to plot lines has been updated to be
     compatible with Matlab's new default color scheme.  The line plot
     color scheme can be set with the axes property "ColorOrder".
 
  ** The default colormap is now set to "viridis" which is also the
     default colormap in matplotlib.  This new colormap fixes some of the
     main issues with the old default colormap "jet" such as its bad
@@ -97,17 +97,17 @@ Summary of important user-visible change
     more Matlab-compatible than the previous m-file version.
 
  ** Dialog boxes--errordlg, helpdlg, inputdlg, listdlg, msgbox,
     questdlg, and warndlg--now exclusively use Qt for rendering.
     Java based versions have been removed.
 
  ** The axes properties "TitleFontSizeMultiplier" and "TitleFontWeight"
     are now implemented which control the default appearance of text
-    created with title() appear.
+    created with title().
     The axes property "LabelFontSizeMultiplier" is now implemented
     which controls the default appearance of text created with
     xlabel(), ylabel(), or zlabel().
 
  ** The graphics property "box" for axes now defaults to "off".
     To obtain equivalent plots to previous versions of Octave use
       set (0, "DefaultAxesBox", "off");
     in your .octaverc file.
@@ -137,20 +137,24 @@ Summary of important user-visible change
     cause Octave to abort execution of a MEX file and return to the
     prompt if an error is encountered in mexCallMATLAB.
 
  ** The MEX API now includes the function mexCallMATLABWithTrap.  This
     function will not abort if an error occurs during mexCallMATLAB, but
     instead will return execution to the MEX function for error
     handling.
 
+ ** The MEX API functions for input validation that begin with "mxIs"
+    (e.g., mxIsDouble, mxIsEmpty, etc.) now return type bool rather than
+    type int.
+
  ** The functions mxAssert and mxAssertS for checking assertions have
     been added.  In order to avoid a performance penalty they are only
     compiled in to debug versions of a MEX file, i.e., that are produced
-    when the '-g' option is given to mex() or mkoctfile().
+    when the '-g' option is given to mex or mkoctfile.
 
  ** Other new MEX API functions include mexEvalStringWithTrap,
     mxIsScalar, mxCreateUninitNumericArray, mxCreateUninitNumericMatrix.
 
  ** Other new functions added in 4.2:
 
       audioformats
       camlight
